docs/REST/authentication.md

```diff
@@ -289,3 +289,68 @@ static class HttpContentExtensions
     }
 }
 ```
+### Java
+```java
+public class ConfigHttpClient {
+
+    public CloseableHttpResponse execute(HttpUriRequest request, String credential, String secret)
+            throws ClientProtocolException, IOException, UnsupportedOperationException, URISyntaxException {
+        Map<String, String> authHeaders = sign(request, credential, secret);
+        authHeaders.forEach(request::setHeader);
+
+        return httpClient.execute(request);
+    }
+
+    private static Map<String, String> sign(HttpUriRequest request, String credential,
+            String secret) throws URISyntaxException, UnsupportedOperationException, IOException {
```

Comment:
NIT: Just for easier to read, can we have `String secret)` on the previous line?